backend/blueprints/spa_api/spa_api.py

```diff
@@ -154,8 +156,12 @@ def api_upload_replays():
         file.seek(0)
         filename = os.path.join(current_app.config['REPLAY_DIR'], secure_filename(file.filename))
         file.save(filename)
-        celery_tasks.parse_replay_task.delay(os.path.abspath(filename))
-    return 'Replay uploaded and queued for processing...', 202
+        result = celery_tasks.parse_replay_task.delay(os.path.abspath(filename))
+        parse_status.append(
+            { 'task_id': result.id,
+            'filename': filename}
+        )
+    return better_jsonify(parse_status), 202
```

Comment:
This returns 
```
[
    {
        "filename": "C:/Users/Harry/Documents/Rocket League/DistributedReplays\\data\\rlreplays\\15D8A37A4FC18658013537A04B68942A.replay",
        "task_id": "d36d0598-eba2-46a2-a013-538eb622f3e3"
    }
]
```
for me.
Why are we returning filename (and if we do, I think `os.path.basename(filename)` would be better.)
What should the client do to find out more about said task_ids though?

Reply:
Yeah good point, we do only need the base name. Will make that change

Like I said above- I'm thinking I'll add another API endpoint which handles the AsyncResult.status operation. The client should send this list of task IDs and it will return a status for each.

Reply:
Well tbh we don't need the filename at all right now- it's just providing some flexibility. 

Should I take it out if we don't have a plan to use it?

Reply:
Yeah I think take it out. Doesn't really have a use. Also I think you don't need better_jsonify here - just jsonify should do it.